content/guides/models/app/features/panels/scatter-plot.md

```diff
@@ -7,15 +7,34 @@ title: Scatter plots
 weight: 40
 ---
 
-Use the scatter plot to compare multiple runs and visualize how your experiments are performing. We've added some customizable features:
+This page explains the use case for scatter plots in W&B, how to create them, and provides an example.
 
-1. Plot a line along the min, max, and average
-2. Custom metadata tooltips
-3. Control point colors 
-4. Set axes ranges
-5. Switch axes to log scale
+## Use case 
```

Comment:
I feel we don't need this heading. Maybe the contents of this section could be moved up into the overview.